### DISCUSSION TITLE
Why are Ernie models gone?

### DISCUSSION THREAD

**Discussion by em020** [langgenius/dify#3266] [Help]
Yesterday I could see there are Ernie model from Baidu, but today when I check again, they are gone, why?

**Comment by em020** [comment]
![image](https://github.com/langgenius/dify/assets/6092370/14ebc23d-1c24-413d-8fab-9a897caa5ad7)

**Comment by em020** [comment]
Found it, it's in model provider, you need to setup key first, then it will be selectable in apps. I'll close this

**Reply by IT-weicheng** [reply]
![image](https://github.com/langgenius/dify/assets/154573082/3804caf0-0fa7-492f-9c4a-ea3f6d244ad7)
 Hello, I have encountered a similar issue as well. Currently, I can only see OpenAI's GPT, the other LLMs are nowhere to be found.

**Reply by IT-weicheng** [reply]
I found it too, thank you. Other models can be found in the settings.
